composer.json

```
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "php": "^7.2|8.0.*|8.1.*",
+        "php": "^7.2|^8.0",
```

@mpdude You can commit on this branch, so feel free to apply if you're really sure you want to take this path and manually add minors as they're released.
```suggestion
        "php": "^7.2|8.0.*|8.1.*|8.2.*|8.3.*",
```

It will also require to add a conditional `--ignore-platform-req=php` in `.github/workflows/tests.yml` to be able to test PHP 8.4 (and so be able to anticipate if any compatibility issue is coming in the next version).

Testing in advance the next version (+ schedule a weekly run for instance) is also a way to be ready pro-actively instead of reactively, and it makes then less useful to be conservative on the PHP compatibility range.